modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/Credential.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.credential;
+
+import org.eclipse.xpanse.modules.credential.enums.CredentialType;
+
+/**
+ * The credential object.
+ */
+public class Credential {
+
+    /**
+     * The name of the credential, this field is provided by the plugins.
+     */
+    String name;
+
+    /**
+     * The userId of the credential.
+     */
+    String userId;

```

Comment:
should this be the description?

Reply:
No, the `userId` is to identify the users if needed. I will add the `description` field for the missing.